regression/tests/helpers.py

```diff
@@ -4,6 +4,19 @@
 import os
 
 
+def get_course_info():
+    """
+    Returns the course info of the course that we use for
+    the regression tests.
+    """
+    return {
```

Comment:
I have changed it to get values from environment variables. Thought default is still set to the course used before.